google/cloud/bigquery/client.py

```diff
@@ -212,7 +225,10 @@ def close(self):
         self._http.close()
 
     def get_service_account_email(
-        self, project=None, retry=DEFAULT_RETRY, timeout=None
+        self,
+        project: str = None,
+        retry: retries.Retry = DEFAULT_RETRY,
+        timeout: float = None,
     ):
```

Comment:
I recall there's a way to annotate the return type, too.

Reply:
Yes, i will specify return type  for all the public methods.